### PR TITLE
ANLcr2

### PR DESCRIPTION
**Pull request by thea-m**
The IDs used in the record refer to the old version of Dǝrsāna Mikāʾel. @CarstenHoffmannMarburg will update them at some point after this PR has gone through.

### CODE REVIEW

File: RomeALincei/ANLcr2.xml [hunk 1]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                                <locus from="7ra" to="18ra"></locus>
-                                <title ref="LIT1295Dersan#Hedar"></title>
+                                <locus from="7ra" to="18ra"/>
+                                <title ref="LIT1295Dersan#Hedar"/>
+                                
                                 <msItem xml:id="ms_i1.1.1">
-                                    <locus from="7ra" to="14rb"></locus>
-                                    <title ref="LIT1295Dersan#HomilyHedar"></title>
+                                    <locus from="7ra" to="14rb"/>
+                                    <title ref="LIT1295Dersan#HomilyHedar"/>
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:60]: HomilyTimothy

File: RomeALincei/ANLcr2.xml [hunk 2]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ንዌጥን፡ በረድኤተ፡ አግዚአብሔር፡ ሕያው፡ ጽሒፈ፡
+                                        ተአምሪሁ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ትንብልናሁ፡ የሃሉ፡ <pb n="15r"/><cb n="a"/> ምስለ፡ ፍቁሩ፡ <space reason="owner"/>
+                                        ለዓለመ፡ ዓለም፡ አሜን። <gap reason="ellipsis"/></incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.3">
-                                    <locus from="15va" to="16rb"></locus>
-                                    <title ref="LIT1295Dersan#MiracleHedar"></title>
+                                    <locus from="15va" to="16rb"/>
+                                    <title ref="LIT1295Dersan#MiracleHedar"/>
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:79]: MiracleSailors

File: RomeALincei/ANLcr2.xml [hunk 3]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                        ትካት፡ ይፌክር፡ <gap reason="ellipsis"/> <cb n="b"/> <gap reason="ellipsis"/>
+                                        ወናሁ፡ እዜንወክሙ፡ በቀዳሚ፡ መዋዕል፡ በባሕር፡ ብዙኃን፡ ሰብእ፡ እምብሔረ፡
+                                        ግብጽ፡ በማዕዶተ፡ ዮርዳኖስ።</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.4">
-                                    <locus from="16rb" to="17vb"></locus>
-                                    <title ref="LIT1295Dersan#CommemorativeHedar">Apparition of St Michael to Joshua</title>
+                                    <locus from="16rb" to="17vb"/>
+                                    <title ref="LIT1295Dersan#CommemorativeHedar">Apparition of St
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:89]: CommemorativeJoshua

File: RomeALincei/ANLcr2.xml [hunk 4]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> አመ፡ ፲ወ፪ ለኅዳር፡ በዛቲ፡ ዕለት፡ ተዝካረ፡ በዓሉ፡
+                                        ለቅዱስ፡ ሚካኤል፡ ወርእሰ፡ ሰማያውያን፡ መሐረ፡ ለዘመደ፡ እጓለ፡ እመሕያው፡ ዘይቀውሙ፡ ኵሎ፡
+                                        ጊዜ፡ ቅድመ፡ መንበሩ፡ ለንጉሠ፡ ሰማይ፡ ወይተንብል፡ በእንቲአነ፡ ወዝ፡ ውእቱ፡ ዘርእዮ፡ ኢያሱ፡
+                                        ወልደ፡ ነዌ፡ በአምሳለ፡ ሐራሁ፡ ለንጉሥ፡ ወደንገፁ፡ እምኔሁ፡</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.1.5">
-                                    <locus from="17vb" to="18ra"></locus>
+                                    <locus from="17vb" to="18ra"/>
                                     <title type="complete" ref="LIT1295Dersan#SalamHedar"/>
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:99]: SalamCompassionate and SalamDesert as far as I can see, now they have become 2 (?)

File: RomeALincei/ANLcr2.xml [hunk 5]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                                <locus from="18ra" to="24va"></locus>
-                                <title ref="LIT1295Dersan#Tahsas"></title>
+                                <locus from="18ra" to="24va"/>
+                                <title ref="LIT1295Dersan#Tahsas"/>
+                                
                                 <msItem xml:id="ms_i1.2.1">
-                                    <locus from="18ra" to="23va"></locus>
-                                    <title ref="LIT1295Dersan#HomilyTahsas"></title>
+                                    <locus from="18ra" to="23va"/>
+                                    <title ref="LIT1295Dersan#HomilyTahsas"/>
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:110]: HomilyDeratewos

File: RomeALincei/ANLcr2.xml [hunk 6]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                
                                 <msItem xml:id="ms_i1.2.2">
-                                    <locus from="23va" to="24ra"></locus>
-                                    <title type="complete" ref="LIT1295Dersan#MiracleTerr">Miracle of a rich man reduced to poverty</title>
-                                    <incipit xml:lang="gez">
-                                        ተአምሪሁ፡  ወሀሎ፡ ፩ብእሲ፡ ክርስቲያናዊ፡ 
-                                    </incipit>                                    
-                                    <note>In <ref type="mss" corresp="BAVet82"></ref> it appears for Ṭərr</note>
+                                    <locus from="23va" to="24ra"/>
+                                    <title type="complete" ref="LIT1295Dersan#MiracleTerr">Miracle
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:117]: @CarstenHoffmannMarburg I do not find this miracle now at all. Please make sure that you may rename things but you may not remove things that have been already referred to multiple times. 
Please restore all deleted items.
Thank you.

File: RomeALincei/ANLcr2.xml [hunk 7]
```diff
@@ -28,337 +29,594 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                        ለሚካኤል፡ ኀበ፡ ያዕቆብ፡ <choice><corr>እስራኤላዊ፡</corr><sic>እስራልዊ፡</sic></choice> እንዘ፡ ይፈርህ፡ እም፡ 
+                                        ኤሳው፡ እኁሁ፡</incipit>
+                                    <note>In <ref type="mss" corresp="BAVet82"/> it appears for
+                                        Ṭərr</note>
+                                </msItem>
+                                
                                 <msItem xml:id="ms_i1.2.4">
-                                    <locus from="24rb" to="24va"></locus>
+                                    <locus from="24rb" to="24va"/>
                                     <title type="complete" ref="LIT1295Dersan#SalamTerr"/>
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml:142]: SalamEldest

Comment (eu-genia) [reply RomeALincei/ANLcr2.xml:142]: etc

**Comment by eu-genia** [comment]
Good, then @CarstenHoffmannMarburg should first go through existing manuscripts and make sure that they can be redescribed with the new structure as it seems to me some elements have disappeared )

**Comment by CarstenHoffmannMarburg** [comment]
> Good, then @CarstenHoffmannMarburg should first go through existing manuscripts and make sure that they can be redescribed with the new structure as it seems to me some elements have disappeared )

I think, it has been done by @eu-genia shortly in https://github.com/BetaMasaheft/Works/pull/1354#pullrequestreview-2500208649.

File: RomeALincei/ANLcr2.xml [hunk 8]
```diff
@@ -170,7 +172,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 
                                 <msItem xml:id="ms_i1.3.3">
                                     <locus from="30rb" to="30vb"/>
-                                    <title>Miracle of the Three Youths in the Furnace</title>
+                                    <title ref="">Miracle of the Three Youths in the Furnace</title>
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml]: Has to be created, i think in the record 1295 there only was a commemoration on the youths

File: RomeALincei/ANLcr2.xml [hunk 9]
```diff
@@ -186,23 +188,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                                         the 12th of Yakkātit</title>
                                     <incipit xml:lang="gez"> በስመ፡ <gap reason="ellipsis"/> ድርሳን፡ ዘቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡
                                         ዘይትነበብ፡ አመ፡ ፲ወ፪ ለየካቲት፡ ትንብልናሁ፡ <gap reason="ellipsis"/> ወሀሎ፡ ፩ ብእሲ፡ ነዳይ፡ ውስተ፡ ዝንቱ፡
                                         ብሔር። ወዓቢይ፡ ሃይማኖቱ፡</incipit>
                                 </msItem>
+                                
                                 <msItem xml:id="ms_i1.4.2">
                                     <locus from="34va" to="35va"/>
-                                    <title ref="LIT1295Dersan#MiracleYakkatit">Miracle of the
+                                    <title ref="">Miracle of the
```
Comment (eu-genia) [inline RomeALincei/ANLcr2.xml]: MiraclParalytic

Comment (CarstenHoffmannMarburg) [reply RomeALincei/ANLcr2.xml]: MiracleParalytic

**Pull request merged**